README.md

```diff
@@ -66,6 +66,8 @@ In order to start up oneview-redfish-toolkit service, there is some mandatory co
 
   * **xml_prettify**: whether XML objects on answers are indented or not
 
+  * **host**: the IP address where redfish service will listen to. Using `host = 0.0.0.0` means to listen to all IP addresses, however it doesn't work on Windows operating systems.
```

Comment:
-"means to listen to"
"means it will listen to"
Complete: On Windows please inform one of the machines IP address, including localhost (127.0.0.1)

Reply:
Once you've found out that it works on Windows, should I write the message as before (```host: the IP address where redfish service will listen to. Using host = 0.0.0.0 means it will listen to all IP addresses```)?

Reply:
As `host = 0.0.0.0` works on Windows, is this pull request still necessary? :thinking: